ElrondNetwork/api.elrond.com
# Update packages

## Description of the reasoning behind the pull request (what feature was missing / how the problem was manifesting itself / what was the motive behind the refactoring)
- Some packages are out of date
  
## Proposed Changes
- Update packages to latest stable version

## How to test
- Check if application is starting
- Check if all app instances listen (cache warmer, queue worker, tx processor)
